product_docs/docs/pgd/5.6/ddl/ddl-command-handling.mdx

```diff
@@ -500,13 +504,6 @@ break the replication due to the `writer` worker throwing the error: `cannot cha
 Generally `CREATE SEQUENCE` is supported, but when using global
 sequences, some options have no effect.
 
-<div id='bdr_ddl_allowed_CreateStmt'></div>
-
-### CREATE TABLE
-
-Generally `CREATE TABLE` is supported, but `CREATE TABLE WITH OIDS` isn't
```

Comment:
Removed this because WITH OIDS was removed in PG12, and we don't support anything older in PGD since PGD 4.0.

Reply:
Got it.